subsys/net/ip/Kconfig.tls

```diff
@@ -10,14 +10,16 @@
 menu "TLS options"
 
 config NET_TLS
-	bool "TCP TLS support"
+	bool
```

Comment:
The original "bool ...." looked much nicer, why was this changed? The "prompt" is kind of redundant keyword.